Tools/autotest/common.py

```diff
@@ -3641,7 +3642,6 @@ def get_parameter_direct(self, name, attempts=1, timeout=60, verbose=True, timeo
                     return m.param_value
                 if verbose:
                     self.progress("(%s) != (%s)" % (m.param_id, name,))
```

Comment:
if you are at it, the verbose #L3635 should be after the check for none

Reply:
Could you PR that, please?